src/dom.js

```diff
@@ -377,11 +293,7 @@ var dom = module.exports = {
 				}
 				// Need to correct for borders if any exist on parent elements
 				if (node !== targetNode) {
-					if (node.currentStyle) {
-						// Oh IE, we do so love working around your incompatibilities
-						borderLeft = parseInt(node.currentStyle.borderLeftWidth, 10);
-						borderTop = parseInt(node.currentStyle.borderTopWidth, 10);
-					} else if (global.getComputedStyle) {
+					if (global.getComputedStyle) {
```

Comment:
That works. Here's the originating PR for reference:

https://github.com/enyojs/enyo/pull/209